src/olympia/reviewers/tests/test_serializers.py

```diff
@@ -101,6 +101,7 @@ def test_requested_file(self):
             'content-script.js', 'icons', 'icons/LICENSE', 'icons/link-48.png',
             'manifest.json'}
 
+        assert data['selected_file'] == 'icons/LICENSE'
```

Comment:
is there a test that checks the default case - that `data['selected_file']` is `manifest.json` (or whatever) when no `file` param is sent?

Reply:
Added one in 45fe5fdffb